tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -738,6 +774,20 @@ public void expectNextElement(ManualSubscriber<T> sub, T expected) throws Interr
       }
     }
 
+    public void expectAnyNextElement(ManualSubscriber<T> sub, Collection<T> expected) throws InterruptedException {
```

Comment:
`Set` extends `Collection`, plus there is no `Set.of` in pre Java-9 but there is `Arrays.asList()`. For the particular test, there are 3 elements to expect at most so `asList` is okay, for any other use, you can call it with a `Set` if you want to.